customer-commons/src/main/java/no/unit/nva/customer/model/CustomerMapper.java

```diff
@@ -0,0 +1,82 @@
+package no.unit.nva.customer.model;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.net.URI;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import nva.commons.utils.JsonUtils;
+
+public class CustomerMapper {
+
+    public static final URI NO_CONTEXT = null;
+    public static URI context;
+
+    static {
+        context = URI.create("https://bibsysdev.github.io/src/customer-context.json");
+    }
+
+    private final ObjectMapper objectMapper;
+    private final String namespace;
+
+    public CustomerMapper(String namespace) {
+        this.namespace = namespace;
+        objectMapper = JsonUtils.objectMapper;
+    }
+
+    /**
+     * Map from Customer from Db to Dto version.
+     *
+     * @param customerDb    customerDb
+     * @return  customerDto
+     */
+    public CustomerDto toCustomerDto(CustomerDb customerDb) {
+        CustomerDto customerDto = objectMapper.convertValue(customerDb, CustomerDto.class);
+        URI id = toId(customerDb.getIdentifier());
```

Comment:
Any reason for why this doesnt live inside the `CustomerDto` ? Wouldn't `CustomerDto` know about `namespace`? Sounds weird if we save identifiers without knowing its related `namespace`?

Reply:
Is it because `Customer` belongs to us (NVA) and this again depends on where it is deployed and environment it is running in probably? Then it makes more sense.